app/views/cohorts/edit.haml

```diff
@@ -18,7 +18,7 @@
   - if can_view_vspdat? # for the moment these are only ever triggered by vspdats
     = f.input :assessment_trigger, collection: @assessment_types, label: 'Assessment submission triggers client addition to cohort:', input_html: {class: :select2}
   = f.input :default_sort_direction, collection: @cohort.class.sort_directions.invert, include_blank: false, selected: @cohort.default_sort_direction, hint: hint, input_html: {class: :select2}
-  = f.input :user_ids, collection: User.active, selected: @cohort.user_viewable_entities.pluck(:user_id), input_html: {multiple: :multiple, class: :select2, style: 'width: 100%;'}, label: 'Grant access to specific users', hint: 'Users specified below who also have "can edit assigned cohorts" or "can view assigned cohorts" will have access this cohort.  Any user with "can manage cohorts" or "can edit cohort clients" access can always access all cohorts.'
+  = f.input :user_ids, collection: User.active, selected: @user_ids, input_html: {multiple: :multiple, class: :select2, style: 'width: 100%;'}, label: 'Grant access to specific users (does not affect groups)', hint: 'Users specified below who also have "can edit assigned cohorts" or "can view assigned cohorts" will have access this cohort.  Any user with "can manage cohorts" or "can edit cohort clients" access can always access all cohorts.'
```

Comment:
How hard would this be to make "Users or Groups" and list all groups here instead of specific users?

Reply:
Not too hard -- moved to using groups, and included the user-specific groups with in the candidates.